RetroBar/Themes/System.xaml

```diff
@@ -963,6 +963,8 @@
                 Value="Stretch" />
         <Setter Property="Margin"
                 Value="4,0,0,0" />
+        <Setter Property="VerticalAlignment"
+                Value="Center"/>
```

Comment:
This isn't correct, the toolbars should keep their existing alignment. However, there is another related change that will need to be made for correctness, that I will comment separately on.

Reply:
Shouldn't tray icons be aligned at top as well, like quick launch?

Reply:
@1280px Yes, but that will require some more extensive theme changes.